R/profile.R

```diff
@@ -200,17 +200,24 @@ prep.step_profile <- function(x, training, info = NULL, ...) {
 
 #' @export
 bake.step_profile <- function(object, new_data, ...) {
-  n <- length(object$profile[[1]])
-  new_data <- new_data[rep(1, n), ]
-  keepers <- c(names(object$columns), names(object$profile))
-  # Keep the predictors in the same order
-  keepers <- names(new_data)[names(new_data) %in% keepers]
-  new_data <- dplyr::select(new_data, !!keepers)
+  col_names <- names(object$columns)
+  profile_names <- names(object$profile)
+  keepers <- c(col_names, profile_names)
 
-  for (i in names(object$columns)) {
-    new_data[[i]] <- rep(object$columns[[i]], n)
+  new_data <- list()
```

Comment:
I found it easier to create a empty list, fill it, order it, tibblify it. Than to modify the existing tibble, order it making sure the lengths matched